Lib/fontTools/ufoLib/converters.py

```diff
@@ -6,16 +6,16 @@
 
 # adapted from the UFO spec
 
-def convertUFO1OrUFO2KerningToUFO3Kerning(kerning, groups):
+def convertUFO1OrUFO2KerningToUFO3Kerning(kerning, groups, glyphSet):
```

Comment:
For maximum bw compatibility, maybe give `glyphSet` a default value of `{}` or `set()`?

Reply:
Sure!